OPHD/States/MapViewState.cpp

```diff
@@ -405,22 +405,22 @@ void MapViewState::onKeyDown(NAS2D::EventHandler::KeyCode key, NAS2D::EventHandl
 	{
 		case NAS2D::EventHandler::KeyCode::KEY_w:
 		case NAS2D::EventHandler::KeyCode::KEY_UP:
-			mMapView->moveView(Direction::North);
+			mMapView->moveView(Direction::NorthWest);
```

Comment:
You know, if we updated the parameter type here, we could change this from:
```cpp
mMapView->moveView(Direction::NorthWest);
```
To:
```cpp
mMapView->moveView(DirectionNorthWest);
```

Same readability, but without needing an enum to offset lookup.

Reply:
Ahh, but there's also the up/down buttons handled by that function.

Reply:
True, but I kinda like the indirection a bit. Kinda like namespaces but without the namespace